hat/assets/js/apps/Iaso/domains/entities/duplicates/details/DuplicateDetails.tsx

```diff
@@ -0,0 +1,284 @@
+import { Box, Divider, Grid, makeStyles, Paper } from '@material-ui/core';
+import { commonStyles, useSafeIntl } from 'bluesquare-components';
+import classnames from 'classnames';
+import React, {
+    FunctionComponent,
+    useCallback,
+    useEffect,
+    useState,
+} from 'react';
+import { useDispatch } from 'react-redux';
+import { isEqual } from 'lodash';
+import TopBar from '../../../../components/nav/TopBarComponent';
+import { TableWithDeepLink } from '../../../../components/tables/TableWithDeepLink';
+import { baseUrls } from '../../../../constants/urls';
+import { useArrayState } from '../../../../hooks/useArrayState';
+import { useObjectState } from '../../../../hooks/useObjectState';
+import { redirectTo } from '../../../../routing/actions';
+import { useDuplicationDetailsColumns } from './hooks/useDuplicationDetailsColumns';
+import {
+    useGetDuplicateDetails,
+    useGetDuplicates,
+} from '../hooks/api/useGetDuplicates';
+import MESSAGES from '../messages';
+import { DuplicateInfos } from './DuplicateInfos';
+import { useDuplicateInfos } from './hooks/useDuplicateInfos';
+import { DuplicateData } from '../types';
+import { useGoBack } from '../../../../routing/useGoBack';
+import { Router } from '../../../../types/general';
+import { DuplicateDetailsTableButtons } from './DuplicateDetailsTableButtons';
+
+type Props = {
+    params: { accountId?: string; entities: string };
+    router: Router;
+};
+
+const useStyles = makeStyles(theme => {
+    return {
+        ...commonStyles(theme),
+        table: {
+            '& .MuiTable-root': {
+                borderLeft: `1px solid rgb(224, 224, 224)`,
+                borderRight: `1px solid rgb(224, 224, 224)`,
+                borderBottom: `1px solid rgb(224, 224, 224)`,
```

Comment:
> Great jobs, but a few remarks:
> 
>     * Please add a `cursor:pointer` on a clickable cell
>       ![Screenshot 2023-03-16 at 11 07 56](https://user-images.githubusercontent.com/12494624/225633891-9a9cea7e-f0cb-4020-b8ec-1aaa9ebaab4a.png)
> 
>     * it looks like a border is missing, also i would move the buttons under the infos, or outside the paper.
>       ![Screenshot 2023-03-16 at 11 07 41](https://user-images.githubusercontent.com/12494624/225634369-5eaf5daf-b809-44d4-a1f1-4f5275aa1343.png)
> 
>     * not sure for this icon
>       ![Screenshot 2023-03-16 at 11 07 29](https://user-images.githubusercontent.com/12494624/225634700-850322f0-6de6-4d87-9451-643b60402923.png)
> 
>     * The filters on the list page can take all the width, the buttons can go under
>       ![Screenshot 2023-03-16 at 11 07 14](https://user-images.githubusercontent.com/12494624/225635017-78db6e30-1e7d-4b50-a836-d8e60c6845a3.png)

- I wouldn't put the buttons under the `Paper` as it would leave tons of blank space. Also it wouldn't be consistent with other pages where such buttons are always on the right side of the screen
- About the filters: I thought the guideline was to follow that pattern:
![Screenshot 2023-03-17 at 13 44 22](https://user-images.githubusercontent.com/38907762/225908114-406de739-c4db-47f0-a828-987e28925a4c.png)
- The icon is the one used for other "compare" features, eg in instances

Reply:
put it in this ticket: https://bluesquare.atlassian.net/browse/IA-1986